ldap/servers/slapd/daemon.c

```diff
@@ -2421,11 +2423,15 @@ netaddr2string(const PRNetAddr *addr, char *addrbuf, size_t addrbuflen)
     return (retstr);
 }
 
-
+/* Create a signal pipe for each listener thread */
 static int
 createsignalpipe(void)
 {
     int i;
+
+    /* Now we know how many listeners there are, setup signalpipes */
+    signalpipes = (signal_pipe*) malloc(sizeof(signal_pipe) * the_connection_table->list_num);
```

Comment:
should use slapi_ch_calloc rather than malloc

Reply:
Yep, fully agreed with tbordaz here. Always use slapi_ch_calloc here, becuase it's actually *faster* than trying to zero something ourselve, and having already zerod blocks of memory is a significant improvement for asan detection but also security.